src/utils/network.ts

```diff
@@ -43,7 +56,35 @@ export function getNetwork(chainId: string): Network {
   }
 }
 
-export enum Networks {
-  MOONBASE = '0x507',
-  MUMBAI = '0x13881',
-}
+export const getBridgeAddress = (networkChainId: NetworksChainID): string => {
+  switch (networkChainId) {
+    case NetworksChainID.GOERLI:
+      return BRIDGE_ADDRESS_GOERLI;
+      break;
+    case NetworksChainID.MOONBASE:
+      return BRIDGE_ADDRESS_MOONBASE;
+      break;
+    case NetworksChainID.MUMBAI:
+      return BRIDGE_ADDRESS_MUMBAI;
+      break;
+    default:
+      throw new Error(`Bridge contract not available for networkChainId`);
+  }
+};
+
+export const getDomainID = (networkChainId: bigint): bigint => {
+  const chainIdHex = web3.utils.toHex(networkChainId) as NetworksChainID;
+  switch (chainIdHex) {
+    case NetworksChainID.GOERLI:
+      return 1n;
+      break;
+    case NetworksChainID.MOONBASE:
+      return 2n;
+      break;
+    case NetworksChainID.MUMBAI:
+      return 3n;
+      break;
+    default:
+      throw new Error(`There is no DomainID for that networkChainId`);
+  }
```

Comment:
no need to use `break` after `return` as function will stop executing on `return`